src/libsystemd/sd-netlink/netlink-types.c

```diff
@@ -22,12 +22,14 @@
 #include <linux/netlink.h>
 #include <linux/rtnetlink.h>
 #include <linux/can/netlink.h>
+#include <linux/fib_rules.h>
 #include <linux/in6.h>
 #include <linux/veth.h>
-#include <linux/if_bridge.h>
-#include <linux/if_addr.h>
 #include <linux/if.h>
 #include <linux/ip.h>
+#include <linux/if_addr.h>
+#include <linux/fib_rules.h>
+#include <linux/if_bridge.h>
```

Comment:
hmm, any reason for this reorder? can you elaborate? usually we try to keep things ordered alphabetically here, except if the kernel hedaers are broken